guides/manual-installation.md

```diff
@@ -693,7 +684,7 @@ facts-service-url: amqp://trento_user:trento_user_password@trento.example.com:56
 api-key: <<TRENTO_API_KEY>>
 ```
 
-> Note: Depending on your setup, adjust the configuration of `/etc/hosts` in order to point the server url https://trento.example.com.
+> **Note:** Depending on your setup, adjust the configuration of `/etc/hosts` in order to point the server url https://trento.example.com.
```

Comment:
I'd add a bit more information here about how this is a hack and that it's expected to have proper name resolution in place. But still, useful for a quick test :+1:

Reply:
I moved this whole part to another document, as the manual installation guide will be included in the official documentation. So we still have the information, in case we need it.